shenyu-integrated-test/shenyu-integrated-test-common/src/main/java/org/apache/shenyu/integratedtest/common/helper/HttpHelper.java

```diff
@@ -48,6 +48,11 @@ public class HttpHelper {
      */
     public static final String GATEWAY_END_POINT = "http://localhost:9195";
 
+    /**
+     * The constant test metrics plugin.
+     */
+    public static final String METRICS_END_POINT = "http://localhost:8090";
```

Comment:
Pls remove this.

Reply:
Ok. I'm still testing it, and I'll clean up the code before the final review.